node/src/BaseClient.ts

```diff
@@ -3630,6 +3632,80 @@ export class BaseClient {
+     * If `number` is specified, returns verbose information limiting the returned PEL entries.
+     * If `0` is specified, returns verbose information with no limit.
+     * @returns A map of detailed stream information for the given `key`. See
+     *     the example for a sample response.
+     * @example
+     * ```typescript
+     * const infoResult = await client.xinfoStream("my_stream");
+     * console.log(infoResult); // Output: {
+     *                          //   length: 2,
+     *                          //   'radix-tree-keys': 1,
+     *                          //   'radix-tree-nodes': 2,
+     *                          //   'last-generated-id': '1719877599564-1',
+     *                          //   'max-deleted-entry-id': '0-0',
+     *                          //   'entries-added': 2,
+     *                          //   'recorded-first-entry-id': '1719877599564-0',
+     *                          //   'first-entry': [ '1719877599564-0', ['some_field", "some_value', ...] ],
+     *                          //   'last-entry': [ '1719877599564-0', ['some_field", "some_value', ...] ],
+     *                          //   groups: 1,
+     *                          // }
+     * ```
+     *
+     * @example
+     * ```typescript
+     * const infoResult = await client.xinfoStream("my_stream", true); // default limit of 10 entries
+     * const infoResult = await client.xinfoStream("my_stream", 15); // limit of 15 entries
+     * console.log(infoResult); // Output: {
+     *                          //   length: 2,
+     *                          //   'radix-tree-keys': 1,
+     *                          //   'radix-tree-nodes': 2,
+     *                          //   'last-generated-id': '1719877599564-1',
+     *                          //   'max-deleted-entry-id': '0-0',
+     *                          //   'entries-added': 2,
+     *                          //   'recorded-first-entry-id': '1719877599564-0',
+     *                          //   entries: [ [ '1719877599564-0', ['some_field", "some_value', ...] ] ],
+     *                          //   groups: [ {
+     *                          //     name: 'group',
+     *                          //     'last-delivered-id': '1719877599564-0',
+     *                          //     'entries-read': 1,
+     *                          //     lag: 1,
+     *                          //     'pel-count': 1,
+     *                          //     pending: [ [ '1719877599564-0', 'consumer', 1722624726802, 1 ] ],
+     *                          //     consumers: [ {
+     *                          //         name: 'consumer',
+     *                          //         'seen-time': 1722624726802,
+     *                          //         'active-time': 1722624726802,
+     *                          //         'pel-count': 1,
+     *                          //         pending: [ [ '1719877599564-0', 'consumer', 1722624726802, 1 ] ],
```

Comment:
```suggestion
     *                          //         'pending': [ [ '1719877599564-0', 1722624726802, 1 ] ],
```

Reply:
quotes are only necessary for keys that have dashes. This is the output produced by VS Code.

Reply:
We still don't have here consumer name

Reply:
for full details, you get the consumer name... according to the shared testes.

Reply:
Yes, but consumer name isn't duplicated per each consumer